README.md

```diff
@@ -1,13 +1,26 @@
 # vsce publish docker action
 
-This action publishes a VS Code extension using vsce.
+This action publishes a VS Code extension using vsce and if needed generate .vsix package.
 
 ## Inputs
 
-`vcse_token`: token to use for authentication (mandatory)
+- `vcse_token`: token to use for authentication (mandatory)
+- `build_pkg`: `yes` or `no`. Default no. If yes .vsix package will be generated and pass set to output
+
+## Outputs
+
+- `vsix_path`: Path of the .vsix package generated
 
 ## Example usage
 
+```yaml
+steps:
+  - name: Publish extension
+    id: publisher
     uses: sigma/vsce-publish-action@v0.0.2
     with:
       vsce_token: ${{ secrets.VSCE_TOKEN }}
+  - name: Process further
```

Comment:
in order for this to do something, shouldn't the prior step set build_pkg?

Reply:
fixed